addon/styles/addon.scss

```diff
@@ -23,6 +23,10 @@ $frost-color-button-primary-active: $frost-color-blue-4; // TODO move to ember-f
   &:focus, &:active {
     outline: none;
   }
+  
+  &:focus {
+    box-shadow: 0px 1px 6px rgba(0, 158, 239, 0.75)
```

Comment:
lol.  I'm like, dammit, the color is spec'd in rgb in the mockup.  Then I convert it and... it's #009EEF.  a/k/a $frost-color-blue-1.  facepalm.